packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,19 +25,61 @@ function forced_termination() {
     exit 1
 }
 
-# Check preconditions for existence of topolvm deployment.
+# Check preconditions for existence of lvms deployment.
 # Adapted from MicroShift code.
 #
 # args: None
-# return: 0 if topolvm readiness should be checked, 1 otherwise
-function topolvmShouldBeDeployed() {
+# return: 0 if lvms readiness should be checked, 1 otherwise
+function lvmsShouldBeDeployed() {
     if ! hash vgs 2>/dev/null; then
         return 1
     fi
-
     if [ -f /etc/microshift/lvmd.yaml ]; then
         return 0
     fi
+    if [ -f /etc/microshift/config.yaml ]; then
+        # The python script checks the config's .storage.driver value. Parsing errors are ignored, these will be caught
```

Comment:
If we'd like to implement a full-fledged python function inline, let's move it out to a separate BASH function at least to help readability.

Why are we not using yq parsing?
```
driver=$(yq e '.storage.driver' /etc/microshift/config.yaml)
case "$driver" in
    "none")
        return 1
        ;;
    "lvms"|"")
        return 0
        ;;
    *)
        return 1
        ;;
esac
```